app/scripts/modules/core/src/diffs/changes.html

```diff
@@ -3,6 +3,24 @@
   <div class="modal-header has-sticky-headers">
     <h3>Changes to {{$ctrl.nameItem.name}}</h3>
   </div>
+  <div class="modal-body" ng-if="(!$ctrl.previousBuildLink || !$ctrl.currentBuildLink) && $ctrl.buildInfo.ancestor && $ctrl.buildInfo.target">
+    <div class="col-md-6">
+      <strong>Previous Build:&nbsp;</strong>{{$ctrl.buildInfo.ancestor}}
+    </div>
+    <div class="col-md-6">
+      <strong>Current Build:&nbsp;</strong>{{$ctrl.buildInfo.target}}
```

Comment:
For consistency, we should include a hash before the build numbers.

Also, is the `&nbsp;` necessary?